README.md

```diff
@@ -25,3 +25,17 @@ Dependencies: Ruby 3.1 and PostgreSQL
 * Run command `bin/rails db:setup` to create database with seed data.
 * Start server: `bin/rails s`
 * Navigate in browser to: http://localhost:3000/admin
+
+
+### Local development setup with Docker
+
+* Clone project using Git\
+* Run `docker build . --tag demo-activeadmin` to create docker image
+* Run `docker volume create demo-activeadmin-db` to create db volume
+* Run Docker
+  - Linux/macOS:  `docker run -v $(pwd):/myapp -v demo-activeadmin-db:/var/lib/postgresql/data -p 3000:3000 -it --rm demo-activeadmin`
+  - Windows CMD:  `docker run -v %cd%:/myapp -v demo-activeadmin-db:/var/lib/postgresql/data -p 3000:3000 -it --rm demo-activeadmin`
+* Run Postgres `service postgresql start`
+* Run command `bin/rails db:setup` to create database with seed data.
+* Start server: `bin/rails s -b 0.0.0.0`
```

Comment:
Rails is smart enough to use the proper version of itself without using a binstub, so we can just say `rails db:setup` and `rails s -b 0.0.0.0`.

Reply:
I've removed binstub in readme.md docker section.